src/beacon/app/routes/devices.py

```
@@ -0,0 +1,310 @@
+from fastapi import APIRouter, Depends, HTTPException, Query
+from sqlmodel import Session, select, func
+from typing import List, Optional, Dict, Any
```

Ruff (0.12.2)

`typing.Dict` imported but unused

Remove unused import

(F401)